Frameworks/UIKit.Xaml/ActivityIndicatorView.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Grid
+    x:Class="UIKit.Xaml.ActivityIndicatorView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UIKit.Xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    x:Name="Root">
+
+    <!-- TODO: Update property names in paths for binding once they have been updated in the header. See #2607 -->
+    <Canvas x:Name="indicatorCanvas"
+            Height="{Binding ElementName=Root, Path=ActivityIndicatorView_Height, Mode=OneWay}"
```

Comment:
So I tried to x:Bind again, and still couldn't manage to get it working for our own properties. It does seem to work fine when binding to an existing property on a control though.

---
In reply to: [113579961](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113579961) [](ancestors = 113579961)